staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go

```diff
@@ -371,7 +371,7 @@ func (b bodyGen) doStruct(sw *generator.SnippetWriter) error {
 `, b.t)
 
 	if len(options) > 0 {
-		sort.Sort(sort.StringSlice(options))
+		sort.Strings(options)
```

Comment:
Seems silly, but I don't actually know that these things are equivalent.   Mind linking me to a golang playground?

Reply:
https://play.golang.org/p/_xl4AUHU_y9